runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/AbstractWindmillStream.java

```diff
@@ -73,9 +73,9 @@ public abstract class AbstractWindmillStream<RequestT, ResponseT> implements Win
   private final Executor executor;
   private final BackOff backoff;
   // Indicates if the current stream in requestObserver is closed by calling close() method
-  private final AtomicBoolean streamClosed;
+  protected AtomicBoolean streamClosed;
```

Comment:
can this be reverted?
It seems at least it should be final, but also setting it as part of stream callback doesn't match the comment

Reply:
done.